pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -94,7 +94,7 @@ public class PersistentDispatcherMultipleConsumers extends AbstractDispatcherMul
     protected volatile PositionImpl minReplayedPosition = null;
     protected boolean shouldRewindBeforeReadingOrReplaying = false;
     protected final String name;
-    protected boolean sendInProgress;
+    private int sendingTaskCounter = 0;
```

Comment:
the variable `sendingTaskCounter` will be accessed by multiple threads, suggestion `AtomicInterger`

Reply:
It simply was unnecessary because sendingTaskCounter is only used within the synchronized block.

Reply:
Yes, you are right